h/services/search_index/_queue.py

```diff
@@ -44,11 +46,28 @@ def add_all(self, annotation_ids, tag, schedule_in=None):
                 name="sync_annotation",
                 scheduled_at=scheduled_at,
                 priority=priority,
-                kwargs={"annotation_id": annotation_id},
+                kwargs={"annotation_id": URLSafeUUID.url_safe_to_hex(annotation_id)},
```

Comment:
The existing `add()` and `add_all()` methods are changed to use the hex IDs like `add_annotations_between_times()` does so that they're all consistent